commands/work/work.go

```diff
@@ -26,50 +21,23 @@ func (cmd) Summary() string {
 
 func (cmd) Usage() string {
 	return `Usage:
-  taskcluster-worker work <config-file> [--logging-level <level>]
-
-Options:
-  -l --logging-level <level>  	Set logging at <level>.
+  taskcluster-worker work <config.yml>
 `
 }
 
 func (cmd) Execute(args map[string]interface{}) bool {
-	// Setup logger
-	var level string
-	if l := args["--logging-level"]; l != nil {
-		level = l.(string)
-	}
-	logger, err := runtime.CreateLogger(level)
+	config, err := worker.LoadConfigFile(args["<config.yml>"].(string))
 	if err != nil {
-		os.Stderr.WriteString(err.Error())
+		fmt.Println(err)
```

Comment:
hmm... I want to do this... but right now I haven't been doing this consistently. Let's fix it separately and make some utility for logging to stderr, this is certainly not the only place we do this wrong.

Notably the qemu-build and run commands...